scout/commands/update/case.py

```diff
@@ -12,8 +12,6 @@
 
 @click.command("case", short_help="Update a case")
 @click.argument("case_id", required=False)
-@click.option("--case-name", "-n", help="Add/update the display name of case")
```

Comment:
Are commands that used to work removed? Or is something else going on? (I asking because I wonder if removing commands will destroy backwards compatibility and make us do a mayor)